app/views/bets/index.html.erb

```diff
@@ -5,5 +5,9 @@
       <h4>This page will display all the existing bets (25/page) and allow the user to use custom search criteria.</h4>
     </div>
   </div>
-  <%= render 'bets/display_bets' %>
+  <div class="d-flex flex-column">
+    <div class="p-2">
+      <%= render 'display_bets', bets: @bets %>
```

Comment:
Why is the intent behind `display_bets` partial ? (versus rendering the table in the index page and rendering individual bets through the `_bet` partial)